File: src/main/java/osha/ConfigureOsha.java
package osha;

import com.gtnewhorizon.gtnhlib.config.Config;

@Config(modid = "osha", filename = "osha")
public class ConfigureOsha {

    @Config.Name("Enable tongs")
    @Config.DefaultBoolean(true)
    @Config.RequiresMcRestart
    public static boolean enableTongs;
}

File: src/main/java/osha/ConfigureOsha.kt
package osha

import com.gtnewhorizon.gtnhlib.config.Config

@Config(modid = "osha", filename = "osha")
object ConfigureOsha {
    @Config.Name("Tongs enabling")
    @Config.DefaultBoolean(true)
    @Config.RequiresMcRestart
    var enableTongs: Boolean = false
}
